dora/underfs/cos/src/main/java/alluxio/underfs/cos/COSUnderFileSystem.java

```diff
@@ -171,6 +185,11 @@ public boolean createEmptyObject(String key) {
 
   @Override
   protected OutputStream createObject(String key) throws IOException {
+    if (mUfsConf.getBoolean(PropertyKey.UNDERFS_OBS_STREAMING_UPLOAD_ENABLED)) {
```

Comment:
why do we read an OBS config here for COS

Reply:
Previously, different UFS used different flags, and the new code I wrote for COS here temporarily used the old switch for testing. But I think it's best to use the same general flag for all Object UFS MPU?